lib/Cargo.toml

```diff
@@ -17,7 +17,7 @@ bitflags = "1"
 camino = "1.0.4"
 chrono = "0.4.19"
 cjson = "0.1.1"
-cap-std-ext = ">= 0.24, <= 0.25"
+cap-std-ext = ">= 0.25"
```

Comment:
I have a pending PR for this I haven't published, but I will; it doesn't look related to this PR though?

Reply:
Oh nevermind right,
```
$ cargo update -p cap-std-ext --precise 0.24.3
$ cargo check
error[E0599]: no method named `atomic_write` found for reference `&ostree::cap_std::fs::Dir` in the current scope
   --> lib/src/container/ocidir.rs:120:13
    |
120 |         dir.atomic_write("oci-layout", r#"{"imageLayoutVersion":"1.0.0"}"#)?;
    |             ^^^^^^^^^^^^ method not found in `&ostree::cap_std::fs::Dir`
```
r

Reply:
It's related in the sense that it's also required for me to get a successful build locally. :) It's in a separate commit:

```
lib: Bump cap-std-ext minimal version to 0.25

We use `atomic_write` which was added in 0.25:
https://github.com/coreos/cap-std-ext/pull/12
```